src/github.py

```diff
@@ -0,0 +1,80 @@
+import os
+import requests
+import streamlit as st
+
+ORG = "NYCPlanning"
+PERSONAL_TOKEN = os.environ["GHP_TOKEN"]
+headers = {"Authorization": "Bearer %s" % PERSONAL_TOKEN}
+BASE_URL = f"https://api.github.com/repos/{ORG}"
+
+
+def parse_workflow(workflow):
+    return {
+        "status": workflow["status"],
+        "conclusion": workflow["conclusion"],
+        "timestamp": workflow["updated_at"],
+        "url": workflow["html_url"],
+    }
+
+
+def get_workflow(repo, name):
```

Comment:
Where is this function called in the code? I don't see it called in the PR, only defined

Reply:
[here](https://github.com/NYCPlanning/data-engineering-qaqc/pull/261/files#diff-39f7e49e9fe3d72dafea43db356c5e896174bfdbb6b2a1c879ca236ef219f28eR47)

Reply:
`ctrl + f`failed me